homeassistant/components/rfxtrx/__init__.py

```diff
@@ -82,7 +84,9 @@ def _bytearray_string(data):
 ]
 
 
-async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+async def async_setup_entry(
+    hass: HomeAssistant, entry: config_entries.ConfigEntry
```

Comment:
Looks like we can clean up the import of `config_entries` if we revert this change and just use `ConfigEntry` which is imported.

Reply:
Will fix.

Reply:
I'll merge here now. We can clean this up later. :+1: